CRM/Financial/Form/FinancialTypeAccount.php

```diff
@@ -299,7 +299,12 @@ public function postProcess() {
       if ($this->_action & CRM_Core_Action::ADD) {
         $params['entity_id'] = $this->_aid;
       }
-      $financialTypeAccount = CRM_Financial_BAO_FinancialTypeAccount::add($params, $ids);
+      try {
+        $financialTypeAccount = CRM_Financial_BAO_FinancialTypeAccount::add($params, $ids);
+      }
+      catch (CRM_Core_Exception $e) {
+        CRM_Core_Error::statusBounce($e->getMessage());
+      }
       CRM_Core_Session::setStatus(ts('The financial type Account has been saved.'));
```

Comment:
I agree with @pradpnayak  - the statusBounce will mean the code does not reach that line, but that will be much clear if you change  it as suggested @jitendrapurohit